inc/mailcollector.class.php

```diff
@@ -1996,10 +2014,9 @@ public function getDecodedContent(\Laminas\Mail\Storage\Part $part) {
          case '7bit':
          case '8bit':
          case 'binary':
+         default:
```

Comment:
In GLPI 9.4, we were returning verbatim content if no transfer encoding was defined. I put back this behaviour as it seems that emails sent by Outlook are not containing this header for text parts (see attached email from #7649).